client/app/scripts/superdesk-ingest/module.js

```diff
@@ -59,6 +59,10 @@ define([
         search: {
             label: 'Search provider',
             templateUrl: 'scripts/superdesk-ingest/views/settings/searchConfig.html'
+        },
+        whatsapp: {
```

Comment:
then you can create a new module and change it there:
```
angular.module('superdesk.ingest.whatsapp', ['superdesk.ingest']).run(function(providerTypes) {
    providerTypes.whatsApp = {...};
});
```
and you have to add this module to list of modules in index.html